.github/workflows/build.yaml

```diff
@@ -22,3 +22,22 @@ jobs:
         jekyll: false
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+    - uses: nelonoel/branch-name@v1.0.1
+    - name: Publish preview
+      uses: jsmrcaga/action-netlify-deploy@v1.5.0
+      with:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
```

Comment:
@alien-mcl please add these two variables as repository secrets

`NETLIFY_SITE_ID` can be found on https://app.netlify.com/sites/hydra-extensions/settings/general (**API ID**)
`NETLIFY_AUTH_TOKEN` can be added on https://app.netlify.com/user/applications

Reply:
Done

Reply:
👌 

https://netlify-previews--hydra-extensions.netlify.app/shacl/